vite.config.ts

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
why did we need to add this ?

Reply:
cause it throwing an eslint error which doesn't make sense 

`'vitest' should be listed in the project's dependencies, not devDependencies.`

Reply:
removed, not needed anymore